site/ja/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,181 @@
+
+この式から次の値が得られます。
+
+- `platform`: プラットフォームまたはオペレーティング システム
+- `major`: ブラウザのメジャー バージョン
+- `mobile`: モバイル デバイス インジケーター
+
+ここで得られた値を、次に示す情報量削減後の形式のテンプレート文字列に挿入します。
+
+```text
+Mozilla/5.0 (${unifiedPlatform[matched.platform]}) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${matched.major}.0.0.0${matched.mobile} Safari/537.36
+````
+
+`unifiedPlatform` の値は、一致する短縮形の `platform` を適切な情報量削減後の値に置き換えることで得られます。
+
+- `Lin` → `Linux; Android 10; K`
+- `Win` → `Windows NT 10.0; Win64; x64`
+- `Mac` → `Macintosh; Intel Mac OS X 10_15_7`
+- `X11; C` → `X11; CrOS x86_64`
+- `X11; L` → `X11; Linux x86_64`
+
+### JavaScript
+
+クライアントサイドの JavaScript で [navigator.userAgent](https://developer.mozilla.org/docs/Web/API/Navigator/userAgent) の現在の値を情報量削減後の形式でオーバーライドするには、次のスニペットを使用します。
+
+デモ: [reduced-ua.glitch.me/javascript.html](https://reduced-ua.glitch.me/javascript.html)
+
+```javascript
+const chromeUAs =
+  /^Mozilla\/5\.0 \(((?<platform>Lin|Win|Mac|X11; C|X11; L)+[^\)]+)\) AppleWebKit\/537.36 \(KHTML, like Gecko\) Chrome\/(?<major>\d+)[\d\.]+(?<mobile>[ Mobile]*) Safari\/537\.36$/;
+const matched = chromeUAs.exec(navigator.userAgent);
+
+if (matched) {
+  // 検出した platform を情報量削減後の値にマッピングする
+  const unifiedPlatform = {
+    Lin: 'Linux; Android 10; K',
+    Win: 'Windows NT 10.0; Win64; x64',
+    Mac: 'Macintosh; Intel Mac OS X 10_15_7',
+    'X11; C': 'X11; CrOS x86_64',
+    'X11; L': 'X11; Linux x86_64',
+  };
+  const reducedUA =
+    `Mozilla/5.0 (${unifiedPlatform[matched.groups.platform]}) ` +
+    `AppleWebKit/537.36 (KHTML, like Gecko) ` +
+    `Chrome/${matched.groups.major}.0.0.0${matched.groups.mobile} Safari/537.36`;
+  // navigator.userAgent を情報量削減後の文字列でオーバーライドする
+  Object.defineProperty(navigator, 'userAgent', {
+    value: reducedUA,
+    writable: false,
+    configurable: true,
+  });
+}
+```
+
+### Node.js
+
+[Node.js](https://nodejs.org/) で受信した [User-Agent ヘッダー](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent)を情報量削減後の形式でオーバーライドするには、次のスニペットを使用します。
+
+デモ: [reduced-ua.glitch.me/server-side](https://reduced-ua.glitch.me/server-side)
+
+```javascript
+const chromeUAs =
+  /^Mozilla\/5\.0 \(((?<platform>Lin|Win|Mac|X11; C|X11; L)+[^\)]+)\) AppleWebKit\/537.36 \(KHTML, like Gecko\) Chrome\/(?<major>\d+)[\d\.]+(?<mobile>[ Mobile]*) Safari\/537\.36$/;
+const matched = chromeUAs.exec(request.get('user-agent'));
+
+if (matched) {
+  const unifiedPlatform = {
+    Lin: 'Linux; Android 10; K',
+    Win: 'Windows NT 10.0; Win64; x64',
+    Mac: 'Macintosh; Intel Mac OS X 10_15_7',
+    'X11; C': 'X11; CrOS x86_64',
+    'X11; L': 'X11; Linux x86_64',
+  };
+  request.headers['user-agent'] = `Mozilla/5.0 (${
+    unifiedPlatform[matched.groups.platform]
+  }) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${
+    matched.groups.major
+  }.0.0.0${matched.groups.mobile} Safari/537.36`;
+}
```

Comment:
Please replicate the original English version as much as possible (except the Japanese translation).

Reply:
I've replicated this section